### PR TITLE
Update Docs: migrate Beta Features - Editorial Workflow

### PR DESCRIPTION
**Pull request by privatemaker**
Continuing migration of Beta Features Documentation #7040

@martinjagodic slightly cleaner this time- last one was made editing within Decap and merging 🥲

### CODE REVIEW

**Comment by privatemaker** [comment]
Or perhaps this should go in the `Workflow` group?

**Comment by martinjagodic** [comment]
I vote for the workflow group.

**Comment by privatemaker** [comment]
Ok. This is now in Workflow Group. I reworked the text a little bit.

**Review by martinjagodic — CHANGES_REQUESTED** [review]
see above

**Comment by privatemaker** [comment]
I removed the whitespace!

**Comment by privatemaker** [comment]
While combing through the docs, I noticed this feature is also mentioned in two other places which cross link to one-another and repeat information which is also (now) getting it's own page with this pull-req.

- [Configuration Options > Publish Mode
](https://decapcms.org/docs/configuration-options/#publish-mode)
- [Add to your Site > Editorial Workflow](https://decapcms.org/docs/add-to-your-site/#editorial-workflow)

I think this should be simplified with both of those sections mentioning and linking to the newly created page.

**Comment by martinjagodic** [comment]
I agree, can you do that as part of this PR?

**Comment by privatemaker** [comment]
Yep. I've done what I think makes most sense in terms of consolidating and trimming. By my eye this is ready for merging.

File: website/content/docs/editorial-workflows.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+From a technical perspective, the workflow translates editor UI actions into common Git commands:
+
+| Actions in Netlify UI     | Perform these Git actions                                                                                         |
+| ------------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| Save draft                | Commits to a new branch (named according to the pattern `cms/collectionName/entrySlug`), and opens a pull request |
+| Edit draft                | Pushes another commit to the draft branch/pull request                                                            |
+| Approve and publish draft | Merges pull request and deletes branch                                                                            |
+
+**Note:** Editorial workflow works with GitHub repositories, and support for GitLab and Bitbucket is [in beta](/docs/beta-features/#gitlab-and-bitbucket-editorial-workflow-support).
```
Comment (martinjagodic) [inline website/content/docs/editorial-workflows.md:23]: This link will not work anymore. We could link to open issues with the EF tag. This way we might get some contributions. 

link: https://github.com/decaporg/decap-cms/issues?q=is%3Aissue+is%3Aopen+label%3A%22area%3A+ui%2Feditorial-workflow%22

**Review by martinjagodic — CHANGES_REQUESTED** [review]
All good except that link.

**Review by martinjagodic — APPROVED** [review]

**Pull request merged**